app/tasks/twitter/listener.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+import time
+import json
+import sys
+import pika
+from tweepy.streaming import StreamListener
+
+class Listener(StreamListener):
+    def __init__(self):
+        #setup rabbitMQ Connection
+    	connection = pika.BlockingConnection(pika.ConnectionParameters(host='localhost'))
```

Comment:
This line is pika (the Python AMQP client) building a connection with the RabbitMQ server. Pika initiates the dispatch below via `channel.basic_publish(...)`.